src/test/java/org/llorllale/cactoos/matchers/MismatchesTest.java

```diff
@@ -62,7 +104,7 @@ public void matches() {
             new Mismatches<>(
                 new TextIs("a"),
                 "Mismatches <a> with message <expected>",
-                ""
+                "Text with value \"a\""
```

Comment:
@baudoliver7 I would expect here to read something related to the assertion exception, such as "Expected: Text with value \"a\"\nbut was: Text is <expected>"

Basically, I would like that the error message from Mismatches tells me what happened and what I was expecting.

I realize this is kind of mind boggling to apply mismatches to itself, but if you try changing the parameters of the usage of Mismatches in the first test of this class, you will see the message you added is not enough to get all the required information :)

Reply:
For example, for the first test, when I change def to abc, I got following error message : Expected: Mismatches <abc> with message <
Expected: Text with value "abc"
 but was: Text is "def">
 but was: Text with value "abc".
Do you think there isn't enough information to get all the required information ?

Reply:
Perhaps, It is best to write this : 
Expected: Mismatches <abc> with message <Text with value "abc" but was: Text is "def">\nbut was: Text with value "abc".

Reply:
@victornoel ping

Reply:
@baudoliver7 Yes, I think the following is ok:
```
Expected: Mismatches <abc> with message <Expected: Text with value "abc" but was: Text is "def">\nbut was: Text with value "abc"
```

Reply:
@victornoel Ok

Reply:
@victornoel Message is Ok now.

Reply:
@baudoliver7 maybe we didn't undertsand each other, I would expect the last argument here to contain something like `Expected: Text with value "expected" but was: Text with value "a"` instead of just `Text with value \"a\"`

Reply:
Aaah Ok ! I get it now.

Reply:
There isn't `\n` just before `but was` ?

Reply:
@victornoel I mean `%n` in place of a space

Reply:
@baudoliver7 feel free to find the best solution :)

Reply:
@victornoel I think It's ok now.

Reply:
@victornoel I sent a commit about what you asked me.